sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneExceptionExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Messaging.EventHubs.Core;
+
+namespace Azure.Messaging.EventHubs.Compatibility
+{
+    /// <summary>
+    ///   The set of extension methods for exception types defined in the
+    ///   Track One client library.
+    /// </summary>
+    ///
+    internal static class TrackOneExceptionExtensions
```

Comment:
I'm sure it is because I am missing a bunch of context, but will track2 use the exceptions? Are they different? Do we need to make the distinction with the name?

Reply:
lol... I had to keep reading the documentation for MapException :P